main.tf

```diff
@@ -8,22 +8,26 @@ terraform {
 
       # A provider alias should be passed for each AWS region.
       # Reference: https://docs.aws.amazon.com/general/latest/gr/rande.html
-      configuration_aliases = [
-        aws.ap-northeast-1, aws.ap-northeast-2, aws.ap-northeast-3,
-        aws.ap-south-1,
-        aws.ap-southeast-1, aws.ap-southeast-2,
-        aws.ca-central-1,
-        aws.eu-central-1,
-        aws.eu-north-1,
-        aws.eu-west-1, aws.eu-west-2, aws.eu-west-3,
-        aws.sa-east-1,
-        aws.us-east-1, aws.us-east-2,
-        aws.us-west-1, aws.us-west-2,
-      ]
+      #configuration_aliases = [
+      #  aws.ap-northeast-1, aws.ap-northeast-2, aws.ap-northeast-3,
+      #  aws.ap-south-1,
+      #  aws.ap-southeast-1, aws.ap-southeast-2,
+      #  aws.ca-central-1,
+      #  aws.eu-central-1,
+      #  aws.eu-north-1,
+      #  aws.eu-west-1, aws.eu-west-2, aws.eu-west-3,
+      #  aws.sa-east-1,
+      #  aws.us-east-1, aws.us-east-2,
+      #  aws.us-west-1, aws.us-west-2,
+      #]
```

Comment:
I don't think you're using this right... This is a module we're suppposed to call it from externally not within this repo